modules/tests/shared/src/test/scala/QueryTest.scala

```diff
@@ -2,7 +2,7 @@
 // This software is licensed under the MIT License (MIT).
 // For more information see LICENSE or https://opensource.org/licenses/MIT
 
-package test
+package tests
```

Comment:
I'm assuming this is a typo as all the other packages here are under 'tests' (though the filesystem path is `test`).